src/request/providers/xhr.ts

```diff
@@ -18,6 +18,7 @@ import SubscriptionPool from '../SubscriptionPool';
  */
 export interface XhrRequestOptions extends RequestOptions {
 	blockMainThread?: boolean;
+	includeRequestedWithHeader?: boolean;
```

Comment:
My only concern with this is that in general we don't have good documentation for any of this, and now we are adding to it.  Without any JSDoc, the API docs won't contain anything as well.  At the very least, we should add JSDoc for options we add, we already have a cleanup task to go back over and review stuff, so lets just not add to the pile of debt.